src/components/dialogs/dynamicsimulation/dynamic-simulation-parameters-selector.jsx

```diff
@@ -147,7 +148,9 @@ const DynamicSimulationParametersSelector = (props) => {
                 </DialogTitle>
                 <DialogContent>
                     <Grid container spacing={2} direction="column" item xs={12} justifyContent={'center'} padding={1}>
-                        <Grid item>{gridItem(mappingNameField, 6)}</Grid>
+                        <Grid item>
+                            <GridItem size={6}>{mappingNameField}</GridItem>
+                        </Grid>
```

Comment:
From Mui doc, a `Grid` must have `container` attribute if it will contains another `Grid`, and `item` attribute if it will be contained by another `Grid`, and the two can be combined.  
So in this case:
```suggestion
                        <Grid container item>
                            <GridItem size={6}>{mappingNameField}</GridItem>
                        </Grid>
```